api/src/main/java/org/pmiops/workbench/db/model/Cohort.java

```diff
@@ -1,17 +1,19 @@
 package org.pmiops.workbench.db.model;
 
+import java.sql.Timestamp;
 import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.Id;
+import javax.persistence.Index;
 import javax.persistence.JoinColumn;
 import javax.persistence.ManyToOne;
 import javax.persistence.Table;
-import org.joda.time.DateTime;
```

Comment:
It doesn't work with Hibernate, or at least, I couldn't figure out how to get it working.

We do have the chance to convert to DateTime in the client representation. So it's not so bad really.